test/playbook.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: all
+
+  roles:
+    - ansible-role-packer
```

Comment:
I think there is a newline at EOF missing ;)

Reply:
done